lib/profileCache.js

```diff
@@ -91,7 +91,7 @@ module.exports = function profileCache(server, options, next) {
         scope: scope
       }}};
       return P.fromCallback(cb => {
```

Comment:
there is a cb here, should that be removed and function below needs to 'return'?

Reply:
we should check if `cache.drop` still uses callbacks or not